pages.de/common/diff.md

```diff
@@ -0,0 +1,27 @@
+# diff
+
+> Vergleiche Dateien und Verzeichnisse.
+
+- Vergleiche Dateien (Listet jene Veränderungen auf, mit denen `alte_datei` zu `neue_datei` wird):
+
```

Comment:
```suggestion
> Mehr Informationen: <https://man7.org/linux/man-pages/man1/diff.1.html>.
```

Reply:
Don't you want to add a newline to the end of the description @navarroaxel?

Reply:
Yep.